x-pack/platform/plugins/shared/index_management/public/application/components/mappings_editor/components/document_fields/fields/fields_list.tsx

```diff
@@ -31,7 +32,12 @@ export const FieldsList = React.memo(function FieldsListComponent({
     return null;
   }
   return (
-    <ul data-test-subj="fieldsList">
```

Comment:
We usually don’t use `HTML` tags directly, as it complicates the maintenance of unified styles and overall makes `UX` support more difficult. We try to make maximum use of `EUI` components. Please use: https://eui.elastic.co/docs/components/display/list-group/

Reply:
I'll just add that it's ideal to use `EuiListGroupItem` instead of `EuiFlexItem` for list items. This will add the necessary `aria-role` attributes like
 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/listitem_role